devtools/aesh/src/main/java/io/quarkus/cli/commands/CreateProjectCommand.java

```diff
@@ -1,66 +1,163 @@
 package io.quarkus.cli.commands;
 
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 import org.aesh.command.Command;
 import org.aesh.command.CommandDefinition;
 import org.aesh.command.CommandResult;
 import org.aesh.command.invocation.CommandInvocation;
+import org.aesh.command.option.Argument;
 import org.aesh.command.option.Option;
+import org.aesh.command.option.OptionList;
+import org.aesh.command.shell.Shell;
 import org.aesh.io.Resource;
+import org.aesh.readline.action.KeyAction;
+import org.aesh.readline.terminal.Key;
+import org.aesh.selector.MultiSelect;
+import org.aesh.selector.SelectorType;
+import org.aesh.terminal.utils.Config;
+import org.apache.commons.lang3.StringUtils;
 
+import io.quarkus.dependencies.Extension;
+import io.quarkus.devtools.commands.AddExtensions;
 import io.quarkus.devtools.commands.CreateProject;
+import io.quarkus.devtools.commands.data.QuarkusCommandException;
+import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.devtools.project.QuarkusProject;
 import io.quarkus.platform.tools.config.QuarkusPlatformConfig;
 
-/**
- * @author <a href="mailto:stalep@gmail.com">Ståle Pedersen</a>
- */
-@CommandDefinition(name = "create-project", description = "Creates a base Quarkus maven project")
+@CommandDefinition(name = "init", generateHelp = true, description = "Creates a base Quarkus project")
```

Comment:
create as we use that everywhere else.

Reply:
Everywhere else is Maven?

Reply:
I renamed it to ```init``` as it was mentioned in a thread recently and I thought it was good. I do not care that much though. I changed it to ```create``` now.